app/uk/gov/hmrc/plasticpackagingtax/registration/views/full_name_page.scala.html

```diff
@@ -48,26 +47,16 @@
     ))
 
     @govukInput(Input(
-        id    = firstNameField.name,
-        name  = firstNameField.name,
-        value = firstNameField.value,
+        id    = fullNameField.name,
```

Comment:
Why don't we set the id to `name` and make the business of identifying the correct field in the acceptance tests that bit cleaner?

Reply:
Again, just following existing patterns used for JobTitle and EmailAdress